internal/runtimehandlerhooks/high_performance_hooks_linux.go

```diff
@@ -1119,7 +1115,24 @@ func calculateMaximalQuota(cpus *cpuset.CPUSet, period uint64) (quota int64, err
 	return
 }
 
-func calculatePodQuota(sharedCpus *cpuset.CPUSet, existingQuota int64, period uint64) (int64, error) {
+func calculatePodQuota(sharedCpus *cpuset.CPUSet, containerManagers []cgroups.Manager, period uint64) (int64, error) {
+	var existingQuota int64
+	var quotaPath string
+	if !node.CgroupIsV2() {
+		quotaPath = "cpu"
+	}
+	for _, mng := range containerManagers {
+		controllerPath := mng.Path(quotaPath)
+		q, err := cgroups.ReadFile(controllerPath, "cpu.cfs_quota_us")
```

Comment:
this isn't present for v2

Reply:
you're right, I was focusing on the v1 fix.

Reply:
Fixed for both